CHANGELOG.md

```diff
@@ -0,0 +1,168 @@
+# Announcing the release of OSMT 2.5.0
+February 28, 2023
+## Enhancements
+OSMT 2.5.0 brings 2 new feature enhancements
+
+- Addition of a "My Workspace" feature, allowing a collection-like place for a user to save and sort RSDs. RSDs can be added and removed the the Workspace, and the Workspace can be converted to a real Collection, and exported as a CSV without creating a real Collection. The Workspace can also be reset (emptied).
```

Comment:
We have double "the" _be added and removed the the Workspace_

Reply:
This was copy/paste from the release announcements in GitHub. It's wrong there too. I'll fix it, thank you.